### PR TITLE
Add ability to add runtime resource presets

### PR DESCRIPTION
**Pull request by dbadura**
**Description**

Changes proposed in this pull request:

- Add ability to set default presets per runtime
- Set profile M for python function in evaluation profile
**Related issue(s)**

See https://github.com/kyma-project/kyma/issues/12461

### CODE REVIEW

File: components/function-controller/pkg/apis/serverless/v1alpha1/function_defaults_test.go [hunk 1]
```diff
@@ -18,6 +20,12 @@ func TestSetDefaults(t *testing.T) {
 	zero := int32(0)
 	one := int32(1)
 	two := int32(2)
+
+	functionProfiles := `
+{
+	"python39": "L"
```
Comment (m00g3n) [inline components/function-controller/pkg/apis/serverless/v1alpha1/function_defaults_test.go:26]: As agreed offline we need to align values with values in sub-chart and chart.

Comment (dbadura) [reply components/function-controller/pkg/apis/serverless/v1alpha1/function_defaults_test.go:26]: I clean up the charts.

**Review by m00g3n — APPROVED** [review]

**Pull request merged**